DataFormats/L1TGlobal/interface/GlobalAlgBlk.h

```diff
@@ -109,9 +101,12 @@ class GlobalAlgBlk
     //Prescale Column
     int m_preScColumn;
 
+    //Menu UUID
+    int m_menuUUID;
+    int m_firmwareUUID;
```

Comment:
uGT data will contain 32-bit hash values generated from menu name and firmware-uuid
They will be generated with the following methods;
tmeventsetup::getMmHashN(L1TUtmTriggerMenu::getName())
tmeventsetup::getMmHashN(L1TUtmTriggerMenu:: getFirmwareUuid())